<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recording</class>
 <widget class="QDialog" name="Recording">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#groupBox  { background-color: #f2f1f5; border-radius: 22px; }
* { color: #3d4150; }
QPushButton { border-style: none;border: 1px solid #DCDFE6;border-radius: 6px;padding: 5px 10px;font-size: 13px; }
QPushButton:hover { color: #b2b7cb; font-weight: bold; background-color: #1296db; }
QScrollBar:vertical { background:#f5f6f8; padding:0px; border-radius: 3px; width:6px; }
QScrollBar::handle:vertical { background:#C0C4CC; border-radius: 3px; }
QScrollBar::handle:vertical:hover { background:#909399; }
QScrollBar::handle:vertical:pressed { background:#909399; }
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical, QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical { background:none; }
QCheckBox, QRadioButton { font-size: 13px; }
QComboBox, QLineEdit, QTextEdit, QTextBrowser, QTimeEdit, QDateEdit, QDateTimeEdit, QSpinBox,
QDoubleSpinBox { background-color: #ffffff; border: 1px solid #DCDFE6; border-radius: 6px; padding: 5px 10px; font-size: 13px; }
QTextEdit, QTextBrowser { padding-right: 3px; }
QLineEdit:hover, QComboBox:hover, QTimeEdit:hover, QDateEdit:hover, QDateTimeEdit:hover, QSpinBox:hover, QDoubleSpinBox:hover { border: 1px solid #409EFF; }
QComboBox::down-arrow { width:10px; height:10px; right:2px; image:url(:/icons/down.png); }
QComboBox::drop-down { subcontrol-origin:padding; subcontrol-position:top right; width:15px; border-left-width:0px; border-left-style:solid; border-top-right-radius:3px; border-bottom-right-radius:3px; border-left-color:#242424; }
QComboBox QAbstractItemView::item{ min-height:20px; min-width:10px; }
QComboBox::drop-down:on{ top: 1px; }
QTimeEdit::up-button, QDateEdit::up-button, QDateTimeEdit::up-button, QSpinBox::up-button, QDoubleSpinBox::up-button { width:10px; height:10px; image:url(:/icons/up.png); padding: 3px 8px 0px 0px; }
QTimeEdit::down-button, QDateEdit::down-button, QDateTimeEdit::down-button, QSpinBox::down-button, QDoubleSpinBox::down-button { width:10px; height:10px; image:url(:/icons/down.png); padding: 0px 8px 3px 0px; }
QTimeEdit::up-button:pressed, QDateEdit::up-button:pressed, QDateTimeEdit::up-button:pressed, QSpinBox::up-button:pressed, QDoubleSpinBox::up-button:pressed { top: -2px; }
QTimeEdit::down-button:pressed, QDateEdit::down-button:pressed, QDateTimeEdit::down-button:pressed, QSpinBox::down-button:pressed, QDoubleSpinBox::down-button:pressed { bottom: -2px; }
QTabWidge::pane{ border:1px solid #DCDFE6; border-radius: 6px; border-top-left-radius: 0px; background-color: #ffffff; selection-background-color:#ffffff; selection-color:#DCDFE6; alternate-background-color:#f2f1f5; gridline-color:#3d4150; }
QTabBa::tab { border:1px solid #DCDFE6; border-top-left-radius: 6px; border-top-right-radius: 6px; font-size: 13px; background-color: #f2f1f5; padding: 2px 6px; }
QTabWidge::pane:top { top:-1px; }
QTabWidgt::pane:bottom { bottom:-1px; }
QTabWidgt::pane:left { right:-1px; }
QTabWidgt::pane:right { left:-1px; }
QTabBr::tab:selected { background-color: #ffffff; border-bottom: none; }
QGroupBox { border: 1px solid #DCDFE6; border-radius: 6px; font-size: 13px; margin-top: 7px; }
QGroupBox::title { subcontrol-origin: margin; subcontrol-position: top center; padding: 0px 5px 3px; }

#widget_menu { background-color: #f8f8f8; border-radius: 16px; }
#btn_playVideo, #btn_playMusic, #btn_ScreenRecording, #toolBtn_screenShot { background-color: #ffffff; border-radius: 5px; padding: 12px 0 0; }
#btn_playVideo:hover, #btn_playMusic:hover, #btn_ScreenRecording:hover, #toolBtn_screenShot:hover,hover { background-color: #f8f8f8; border: 1px solid #f0f0f0; }
#lable_title { padding: 0 10px; }
#btn_language, #btn_meeting, #btn_normal, #btn_notion { padding: 8px 12px; color: #d1d8e7;  border: none; }
#btn_language:hover, #btn_meeting:hover, #btn_normal:hover, #btn_notion:hover { background-color: #ffffff; border-radius: 15px; color: #b8c3ce; }
#btn_personal_homePage, #btn_quit, #btn_scaledToSmall, #btn_littleshow {  border: none; }
#btn_personal_homePage:hover, #btn_quit:hover, #btn_scaledToSmall:hover { background-color: #eeeff4; border-radius: 10px; }
#stackedWidget { background-color: #f8f8f8; border-radius: 16px; }
#page_memberShip, #page_normal, #page_notion, #page_words { color: #C0C4CC; }



</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>461</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>录制设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QSpinBox" name="spinBox_frame_rate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>20</number>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_size">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1920X1080</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2560X1440</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2560X1600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1280X720</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>720X480</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="combo_qulity">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>中等质量</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>低质量</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>高质量</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_frame_rate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>录制帧率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_recording_size">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>录制尺寸</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_recording_qulity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>录制质量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_startRecording">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开始录屏</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_path">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>录制路径:</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>录制时间:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resMain.qrc">
      <normaloff>:/images/UI/cha.png</normaloff>:/images/UI/cha.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resMain.qrc"/>
 </resources>
 <connections/>
</ui>
